### ISSUE TITLE
Allow both aggregate and simple selectors in the same SELECT statement

### ISSUE THREAD

**Issue by dekimir** [scylladb/scylla#4447]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): a66a5d423abd0fc3687fefc4a069c2f0827fcf1a
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): Fedora 29

## Description
Cassandra allows mixing aggregate and simple selectors, while ScyllaDB does not.

## Example
Cassandra:
```cql
cqlsh:ks> create table t(p int, c int, n int, primary key((p), c));
cqlsh:ks> select sum(n), p from t;

 system.sum(n) | p
---------------+------
             0 | null
```

ScyllaDB:
```cql
cqlsh:ks> create table t(p int, c int, n int, primary key((p), c));
cqlsh:ks> select sum(n), p from t;
InvalidRequest: Error from server: code=2200 [Invalid query] message="the select clause must either contains only aggregates or none"
```

**Comment by dekimir** [comment]
Relevant code excerpt: https://github.com/scylladb/scylla/blob/master/cql3/selection/selection.cc#L151

**Comment by tzach** [comment]
@dekimir thanks for reporting

**Comment by ptgoetz** [comment]
We are hitting this.
+1 for getting this fixed + back ported.
We are currently on `3.0.5-0.20190414.403f66eca`

**Comment by avikivity** [comment]
Unfortunately, the fix requires #2206 which is too large to risk a backport.

It is possible to backport #4447 alone, but without the additional behavior that the non-aggregate selection be the first row. Would that work for you?

**Comment by eyalgutkind** [comment]
@avikivity @tzach Do we have a patch for the above issue ?
